docs/config.json

```diff
@@ -644,7 +644,11 @@
             {
               "title": "Terraform Provider",
               "slug": "/management/dynamic-resources/terraform-provider/"
-            }
+            },
+	    {
+	      "title": "Import Teleport Resources into Terraform",
+	      "slug": "/management/dynamic-resources/import-terraform-resources/"
+	    }
```

Comment:
It's not related to this PR per-se, but I've always found our terraform documentation to be buried and non-discoverable. Whenever I go to search something terraform-provider-related, I always get confused where I should look (is it managing cluster? deploying agents? reference?), and can never find what I'm looking for "organically" and end up just using search.

What do think about elevating all Terraform Provider relevant documentation to its own top-level section in the left nav bar? In fact, I would probably want to do the same for Kube Operator later. I think we had both of them at the top level long time ago but then they got buried during some docs restructuring.

Reply:
In the past, these guides were in `docs/pages/management/guides`, which was kind of a catch-all section with no explicit theme.

In the current docs, we put guides related to dynamic resources in `/management/dynamic-resources`. Reference guides are mostly but not always in "Reference", and guides related to deploying agents are in `/agents`. Since the "Deploy Agents with Terraform" guide has to do with agents, and not dynamic resources, I put it in the `/agents` section.

I can see how this would be confusing if you're looking for all content related to Terraform. In general, the docs are [inconsistent](https://github.com/gravitational/teleport/issues/24120) in where they store reference information, so I'm not sure of a good solution for the reference guide without taking more time to look at this.

On my 4k monitor at the default zoom, the left sidebar already extends past the bottom of the viewport—I'm also not sure if we can accommodate more sidebar sections, but it might be worth looking into. Would it make sense to promote the "Dynamic Resources" docs to a top-level section that includes content for the Kubernetes operator and Terraform provider?

Reply:
I can't for the life of me navigate around the docs hierarchy myself. I'll typically just ask AI bot for pointers rather than hunting the menus. I'd really appreciate if you could simply `cmd-f` to find strings in there... but the menu items are folded unless you are visiting a page deeper in the hierarchy.